### PR TITLE
ceph health check at the end of deploymnet TC

### PR DESCRIPTION
**Pull request by vavuthu**
There are case where ceph health check failed during teardown but its recovered after some
retries. To avoid deployment failures, we need to have health check at the end of
deployment test case

Fixes: #5921 

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Comment by petr-balogh** [comment]
Do we have understanding why it's happening on the fresh deployment? Is this something which can happen on freshly installed cluster? 

Have you discussed with devs if this: `Health: HEALTH_WARN Reduced data availability: 1 pg peering` is something we can expect on the fresh deployment?

As it might be some regression if we didn't see it before. I would like to confirm with devs that this is really OK to see on fresh deployment.

By default it's has retries and delays set to try retry for like 10 mins.  Are we ok with this timeout? Or do we want to shorten this to lower number?
@vavuthu

**Comment by vavuthu** [comment]
verified here: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/4257/console

```
2022-05-19 12:15:28  06:45:28 - MainThread - ocs_ci.helpers.sanity_helpers - INFO - Deleting resources as a sanity functional validation
2022-05-19 12:15:28  06:45:28 - MainThread - tests.ecosystem.deployment.test_deployment - INFO - Verifying ceph health after deployment
2022-05-19 12:15:28  06:45:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc wait --for condition=ready pod -l app=rook-ceph-tools -n openshift-storage --timeout=120s
2022-05-19 12:15:28  06:45:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}'
2022-05-19 12:15:29  06:45:29 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage exec rook-ceph-tools-5ffb6bbf8c-ll25l -- ceph health
2022-05-19 12:15:30  06:45:29 - MainThread - ocs_ci.utility.utils - INFO - Ceph cluster health is HEALTH_OK.
2022-05-19 12:15:30  PASSED
```

**Comment by vavuthu** [comment]
> Do we have understanding why it's happening on the fresh deployment? Is this something which can happen on freshly installed cluster?

We didn't see this issue on every deployment ( atleast I saw this issue 2 times ) and yes, its on fresh deployment.
> Have you discussed with devs if this: `Health: HEALTH_WARN Reduced data availability: 1 pg peering` is something we can expect on the fresh deployment?
> 
> As it might be some regression if we didn't see it before. I would like to confirm with devs that this is really OK to see on fresh deployment.
> 
> By default it's has retries and delays set to try retry for like 10 mins. Are we ok with this timeout? Or do we want to shorten this to lower number? @vavuthu

**Review by petr-balogh — APPROVED** [review]

**Review by jimdevops19 — APPROVED** [review]
